Agents/GFAAgent/docker/.m2/settings-security.xml

```diff
@@ -0,0 +1,3 @@
+<settingsSecurity>
+  <master>MASTER_PASSWORD</master>
+</settingsSecurity>
```

Comment:
Please move all current folders in the docker directory to the root directory. Usually, .m2 and credentials are at the root level. The entrypoint.sh is up to you as that is unique